### PR TITLE
Adjust JIT automated testing for tiered compilation

### PR DESCRIPTION
**Pull request by noahfalk**

### CODE REVIEW

**Comment by noahfalk** [comment]
@dotnet-bot help

**Comment by noahfalk** [comment]
@dotnet-bot test ci please

**Comment by noahfalk** [comment]
@RussKeldorph

File: netci.groovy [hunk 1]
```diff
@@ -237,6 +241,8 @@ class Constants {
                // 'illink'
                'r2r_jitstress1':                         ["R2R_FAIL", "R2R_EXCLUDE", "JITSTRESS_FAIL", "JITSTRESS_EXCLUDE"],
                'r2r_jitstress2':                         ["R2R_FAIL", "R2R_EXCLUDE", "JITSTRESS_FAIL", "JITSTRESS_EXCLUDE"],
+               'r2r_jitstress1_tiered':                  ["R2R_FAIL", "R2R_EXCLUDE", "JITSTRESS_FAIL", "JITSTRESS_EXCLUDE"],
```
Comment (BruceForstall) [inline netci.groovy]: You should also add jitstress1_tiered/jitstress2_tiered in this list (validArmWindowsScenarios).

Comment (noahfalk) [reply netci.groovy]: 👍

**Comment by noahfalk** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test

**Comment by noahfalk** [comment]
@dotnet-bot help

**Comment by noahfalk** [comment]
test ci please

**Comment by noahfalk** [comment]
@dotnet-bot test Windows_NT x64 Checked CoreFX Tests

**Pull request merged**